airsenal/framework/utils.py

```diff
@@ -43,6 +43,7 @@ def get_max_gameweek(season=CURRENT_SEASON, dbsession=session):
     max_gw_fixture = (
         dbsession.query(Fixture)
         .filter_by(season=season)
+        .filter(Fixture.date.isnot(None))
```

Comment:
I'd prefer this filter to also be on the gameweek value rather than the fixture date:

```suggestion
        .filter(Fixture.gameweek.isnot(None))
```